index.js

```diff
@@ -6,17 +6,19 @@
 var CSSescape = require('css.escape')
 
 exports.hash = function hash (h, options) {
-  if (!h) return
   options = options || {}
+  var cb = options.callback || function () {}
+  // There's no hash to scroll to, so "success", we did it.
+  if (!h) return void cb(true)
```

Comment:
Can we invert the signature of the callback so we pass `true` when there's an error, and no arguments when success? This brings it into alignment with Node callback conventions.

Reply:
ah, right. yeah. I'll do that 😄

Reply:
Inverted. Good catch.